src/android/ConnectPlugin.java

```diff
@@ -701,6 +707,23 @@ private void executeLogin(JSONArray args, CallbackContext callbackContext) throw
         }
     }
 
+    private void enableHybridAppEvents() {
+        try {
+            Context appContext = cordova.getActivity().getApplicationContext();
+            Resources res = appContext.getResources();
+            int enableHybridAppEventsId = res.getIdentifier("fb_hybrid_app_events", "bool", appContext.getPackageName());
+            boolean enableHybridAppEvents = enableHybridAppEventsId != 0 && res.getBoolean(enableHybridAppEventsId);
```

Comment:
> Add some null check to the code in case the parameter would not be defined at all (shit happens you know)
```
enableHybridAppEventsId != 0
``` 
part does that for android